README.md

```diff
@@ -1,5 +1,6 @@
 
-# UppHub Official Registry
+# UppHub
+---------
```

Comment:
Not sure it should stay here. "-------" if it is not use in ultimatepp the we should not be there. Style consistent first.

Reply:
Ok, but that's just  displays horizontal ruler on markdown.

Reply:
@klugier 

Removed it as unnecessary.